change/@fluentui-react-input-c5b897ac-9f07-4c14-8dcf-16702a2e9e1a.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Publish react-input",
```

```suggestion
  "comment": "Initial release",
```

_nit_ Feel free to change anything else, but "Publish react-input" under "react-input" section does not have much sense 🙄